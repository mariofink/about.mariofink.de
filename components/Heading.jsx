const Heading = ({ children }) => (
  <h1 className="text-2xl sm:text-6xl font-bold">{children}</h1>
);

export default Heading;
