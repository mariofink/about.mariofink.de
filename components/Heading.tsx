interface props {
  children: React.ReactNode;
}

const Heading = ({ children }: props) => (
  <h1 className="text-2xl sm:text-6xl font-bold">{children}</h1>
);

export default Heading;
