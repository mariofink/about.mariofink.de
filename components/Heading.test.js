import React from "react";
import renderer from "react-test-renderer";
import Heading from "./Heading";

describe("Heading", () => {
  it("renders the main page heading", () => {
    const tree = renderer.create(<Heading>My Heading</Heading>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
