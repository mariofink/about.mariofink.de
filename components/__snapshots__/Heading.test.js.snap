// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders the main page heading 1`] = `
<h1
  className="text-2xl sm:text-6xl font-bold"
>
  My Heading
</h1>
`;
